src/app/carbon-estimation/carbon-estimation.constants.ts

```diff
@@ -7,80 +7,102 @@ export enum EmissionsColours {
   Downstream = '#4B7E56',
 }
 
+export enum PlaceholderEmissionsColours {
+  Upstream = '#333',
+  Direct = '#666',
+  Indirect = '#888',
+  Downstream = '#AAA',
+}
```

Comment:
I haven't tried to match the greyscale for the placeholder chart to the shades in the actual chart.

Reply:
I think that's fine, doing it from the real colours might not actually give enough difference between the colours to make them identifiable.